packages/design-tokens/tsconfig.json

```diff
@@ -3,8 +3,14 @@
   "compilerOptions": {
     "resolveJsonModule": true,
     "esModuleInterop": true,
-    "types": ["node", "jest"]
+    "jsx": "react",
+    "types": ["react", "node", "jest"]
   },
-  "include": ["src/**/*.ts", "src/**/*.json"],
+  "include": [
+    "src/**/*.ts",
+    "src/**/*.json",
+    "stories/**/*.tsx",
+    "stories/**/*.ts"
```

Comment:
Includes stories folder in typescript checks but does not include them in build because we are excluding storybook files in `tsconfig.packages.build.json`